.github/ISSUE_TEMPLATE/flaky_test_report.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Flake Report
+about: Report a flaky test in KubeVirt
+title: 'test_name'
+labels: kind/flake
+assignees: ''
+---
+
+# What happened
+
+<!-- insert test name -->
+Flaky test detected: {test_name} [1]
+
+<!-- labels for flaky tests -->
+/kind flake
+/priority critical-urgent
+
+<!-- sig assignment
+     all tests contain a sig identifier, please assign the corresponding SIG to the issue, i.e.
+     for a test name containing [sig-compute] or [sig-operator]
+-->
+/sig compute
+
+<!-- note: the flakefinder url needs to be a stable one, i.e. instead of the moving latest report use any with a date instead -->
+[1]: {flakefinder_url}
+
+## Additional context
+Add any other context about the problem here.
+
+# Flake Action Plan
+
+**As the assignee**, **thoroughly review the issue** and put the resulting report as comment into this issue.
+
+## Then decide on one of the following actions:
+
+### **The flake is a critical bug**
```

Comment:
```suggestion
### **The flake is a bug**
```
I think the actions below should be done for any bug found through a flake.


Also should we have a section for `an issue with the test`  - some flakes have been related to the test not doing what it should - not sure if this would fall under a bug for kubevirt.

Reply:
Good point, I agree!